src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11838,19 +11762,14 @@ private ArgumentSyntax ParseArgumentExpression(bool isIndexer)
             }
             else
             {
-                if (refKindKeyword?.Kind == SyntaxKind.InKeyword)
-                {
-                    refKindKeyword = this.CheckFeatureAvailability(refKindKeyword, MessageID.IDS_FeatureReadOnlyReferences);
-                }
-
                 // According to Language Specification, section 7.6.7 Element access
                 //      The argument-list of an element-access is not allowed to contain ref or out arguments.
                 // However, we actually do support ref indexing of indexed properties in COM interop
                 // scenarios, and when indexing an object of static type "dynamic". So we enforce
                 // that the ref/out of the argument must match the parameter when binding the argument list.
 
-                expression = (refKindKeyword?.Kind == SyntaxKind.OutKeyword)
-                    ? ParseExpressionOrDeclaration(ParseTypeMode.Normal, feature: MessageID.IDS_FeatureOutVar, permitTupleDesignation: false)
+                expression = refKindKeyword?.Kind == SyntaxKind.OutKeyword
+                    ? ParseExpressionOrDeclaration(ParseTypeMode.Normal, feature: 0, permitTupleDesignation: false)
```

Comment:
>0 [](http://example.com/codeflow?start=82&length=1)

MessageID.None?

Reply:
this follows the pattern of all otehr callers.